litellm/__init__.py

```diff
@@ -611,6 +615,23 @@ def add_known_models():
     "empower/empower-functions-small",
 ]
 
+snowflake_models = [
```

Comment:
instead of hardcoding this, can you just add it to our model_prices_and_Context_window.json map 

along with the pricing + max token information, this will make it easier for users to find it - https://models.litellm.ai/